Source/Extensions/Blazorise.SignaturePad/SignaturePad.razor.cs

```diff
@@ -0,0 +1,283 @@
+
+    /// <summary>
+    /// Gets or sets the event that is triggered when the signature pad value changes. The event provides the new signature as a byte array.
+    /// </summary>
+    [Parameter] public EventCallback<byte[]> ValueChanged { get; set; }
+
+    /// <summary>
+    /// Gets or sets the event that is triggered when a stroke ends on the signature pad. The event provides the signature pad's current image data as a PNG-encoded Data URL.
+    /// </summary>
+    [Parameter] public EventCallback EndStroke { get; set; }
+
+    /// <summary>
+    /// Gets or sets the event that is triggered when a new stroke begins on the signature pad. The event provides information about the starting point of the stroke.
+    /// </summary>
+    [Parameter] public EventCallback BeginStroke { get; set; }
+
+    /// <summary>
+    /// The radius of a single dot. Also the width of the start of a mark.
+    /// </summary>
+    [Parameter] public double DotSize { get; set; }
+
+    /// <summary>
+    /// The minimum width of a line.
+    /// </summary>
+    /// <value>The minimum width.</value>
+    [Parameter] public double MinLineWidth { get; set; }
+
+    /// <summary>
+    /// The maximum width of a line.
+    /// </summary>
+    /// <value>The maximum width.</value>
+    [Parameter] public double MaxLineWidth { get; set; }
+
+    /// <summary>
+    /// The time in milliseconds to throttle drawing. Set to 0 to turn off throttling.
+    /// </summary>
+    [Parameter] public int Throttle { get; set; }
+
+    /// <summary>
+    /// The minimum distance between two points to add a new point to the signature.
+    /// </summary>
+    [Parameter] public int MinDistance { get; set; }
+
+    /// <summary>
+    /// The color used to clear the background. Can be any color format accepted by context.fillStyle.
+    /// </summary>
+    [Parameter] public string BackgroundColor { get; set; }
+
+    /// <summary>
+    /// The color used to draw the lines. Can be any color format accepted by context.fillStyle.
+    /// </summary>
+    [Parameter] public string PenColor { get; set; }
+
+    /// <summary>
+    /// The weight used to modify new velocity based on the previous velocity.
+    /// </summary>
+    [Parameter] public double VelocityFilterWeight { get; set; }
+
+    /// <summary>
+    /// The image type [png, jpeg, svg] to get from the canvas element.
+    /// </summary>
+    [Parameter] public SignaturePadImageType ImageType { get; set; } = SignaturePadImageType.Png;
+
+    /// <summary>
+    /// The encoder options for image type [png, jpeg] to get from the canvas element.
+    /// </summary>
+    [Parameter] public double? ImageQuality { get; set; }
+
+    /// <summary>
+    /// The encoder options for image type [svg] to get from the canvas element.
+    /// </summary>
+    [Parameter] public bool IncludeImageBackgroundColor { get; set; }
+
+    /// <summary>
+    /// If true, prevents the user interactions and make it appear normal.
```

Comment:
make it appear normal, is redudant, it just does nothing layout wise.
`If true prevents the user interactions.`?